tests/circuit_synthesis/test_simulation.py

```
@@ -127,6 +127,23 @@ def test_lut(steane_code: CSSCode) -> None:
     assert np.sum(estimate_3) == 0
 
 
+@pytest.mark.parametrize("print_progress", [False, True])
+def test_lut_chunked_parallel(print_progress: bool) -> None:
+    """Test that the multi-chunk (parallel) LUT generation agrees with the single-chunk one."""
+    checks = np.array([[1, 1, 0], [0, 1, 1]], dtype=np.int8)
+
+    serial_lut = LutDecoder._generate_lut(checks)  # ruff: ignore[private-member-access]
```

Ruff (0.16.3)

[warning] Invalid rule code in suppression: private-member-access

Enable `lint.preview` to use rule names

(RUF102)